test/test.js

```diff
@@ -1,133 +1,259 @@
 var expect = require('chai').expect,
     scheme = require('..');
 
-describe('default', function() {
+describe('path', function() {
+    describe('default', function() {
+        it('should return path + tech', function() {
+            expect('a/a.js')
+                .eql(scheme().path({ block: 'a' }, 'js'));
```

Comment:
Oh, you are right! And it's really strange 'cos I don't like the way it's done and I remember that when it was written intuitively `actual` and `expected` were swapped in error messages.
Will toggle them, thanks!